packages/api/src/external/carequality/document/get-non-existent-doc-refs.ts

```diff
@@ -49,7 +50,8 @@ async function checkDocRefsExistInS3(
     documents,
     async doc => {
       try {
-        const fileName = createFileName(cxId, patientId, doc.metriportId);
+        const fileExtension = getFileExtension(doc.contentType || undefined);
```

Comment:
Either we set or undefined or we need to update `getFileExtension` to accept null

Reply:
We should prob be using `createDocumentFilePath`, no?

If for some reason no, at least we need to pass the full filename as 3rd param to `createFilePath` - see implementation of `createDocumentFilePath()`.